packages/zend-ldap/library/Zend/Ldap.php

```diff
@@ -982,15 +1001,16 @@ public function search($filter, $basedn = null, $scope = self::SEARCH_SCOPE_SUB,
                 $search = @ldap_search($this->getResource(), $basedn, $filter, $attributes, 0, $sizelimit, $timelimit);
                 break;
         }
-
+        
         if($search === false) {
             /**
              * @see Zend_Ldap_Exception
              */
             // require_once 'Zend/Ldap/Exception.php';
             throw new Zend_Ldap_Exception($this, 'searching: ' . $filter);
         }
-        if ($sort !== null && is_string($sort)) {
+        // ldap_sort: This function has been DEPRECATED as of PHP 7.0.0 and REMOVED as of PHP 8.0.0. Relying on this function is highly discouraged.
+        if (PHP_VERSION_ID < 70000 && $sort !== null && is_string($sort)) {
```

Comment:
is there any alternative to `ldap_sort` for PHP 7+?

Reply:
Maybe this could be ported https://github.com/laminas/laminas-ldap/commit/35162e69f2685400470bd5d67764050bc521011e

Reply:
#168